codec/types/any_test.go

```diff
@@ -0,0 +1,53 @@
+package types
```

Comment:
```suggestion
package types_test
```

please

Reply:
I can't because I need to test private fields. That was the main reason I had to create the `any_test.go` file.

Reply:
I see. So please rename the file to `any_internal_test.go` (there are other examples in the `types` package) and everything should be sorted :+1:

Reply:
good idea. done.